runelite-client/src/main/java/net/runelite/client/ui/overlay/components/ImagePanelComponent.java

```diff
@@ -46,6 +46,18 @@
 	private static final int BOTTOM_BORDER = 4;
 	private static final int SEPARATOR = 1;
 
+	private final int wrapAfter;
```

Comment:
Adding new constructor or modifying ImagePanelComponent is wrong way to do this.

Reply:
Would it be preferred if I extended it and override the render method?

Reply:
I mostly disagree with any modifications of ImagePanelComponent, because the component itself is wrong and should be merged with PanelComponent.

Reply:
What approach would be preferable here in that case?

Reply:
I created #2405 to resolve this issue.

Reply:
Okay the pull has been merged, so now I guess you can add support for wrapping for PanelComponent, what would be useful for other stuff too. Make sure to support both Horizontal and Vertical alignment.